chain/x/zoracle/internal/types/key.go

```diff
@@ -110,3 +112,9 @@ func GetValidatorAddressAndExternalID(
 	externalID := ExternalID(binary.BigEndian.Uint64(externalIDBytes))
 	return key[prefixLength+16:], externalID
 }
+
+func ReporterStoreKey(validatorAddress sdk.ValAddress, reporterAddress sdk.AccAddress) []byte {
```

Comment:
Move this function to the location it belongs to.
Add comment
`ReporterStoreKey is a function to generate key for each validator-reporter pair in store.`